pcawg_metadata_parser/parse_gnos_xml.py

```diff
@@ -431,7 +435,12 @@ def create_vcf_entry(donor_unique_id, analysis_attrib, gnos_analysis, annotation
     workflow_name = vcf_entry.get('workflow_details').get('variant_workflow_name')
     workflow_version = vcf_entry.get('workflow_details').get('variant_workflow_version')
 
-    if workflow_name == 'SangerPancancerCgpCnIndelSnvStr' and (( workflow_version.startswith('1.0.') or workflow_version.startswith('1.1.'))
+    if workflow_name == 'SangerPancancerCgpCnIndelSnvStr+SVFIX' and (( workflow_version.startswith('1.0.') or workflow_version.startswith('1.1.'))
+            and not workflow_version in ['1.0.0', '1.0.1']):
+        vcf_entry['vcf_workflow_type'] = 'sanger'
+        vcf_entry['vcf_workflow_status'] = 'fixed'
```

Comment:
Well. This is good point. So basically all the previous versions are v1 version. Fixed version are v2 version. Correct?




On Jan 11, 2016, at 3:34 PM, Junjun Zhang <notifications@github.com<mailto:notifications@github.com>> wrote:


In pcawg_metadata_parser/parse_gnos_xml.py<https://github.com/ICGC-TCGA-PanCancer/pcawg-central-index/pull/55#discussion_r49374703>:

> @@ -431,7 +435,12 @@ def create_vcf_entry(donor_unique_id, analysis_attrib, gnos_analysis, annotation
>      workflow_name = vcf_entry.get('workflow_details').get('variant_workflow_name')
>      workflow_version = vcf_entry.get('workflow_details').get('variant_workflow_version')
>
> -    if workflow_name == 'SangerPancancerCgpCnIndelSnvStr' and (( workflow_version.startswith('1.0.') or workflow_version.startswith('1.1.'))
> +    if workflow_name == 'SangerPancancerCgpCnIndelSnvStr+SVFIX' and (( workflow_version.startswith('1.0.') or workflow_version.startswith('1.1.'))
> +            and not workflow_version in ['1.0.0', '1.0.1']):
> +        vcf_entry['vcf_workflow_type'] = 'sanger'
> +        vcf_entry['vcf_workflow_status'] = 'fixed'


If we use vcf_workflow_result_version, you can actually populate 'v1' for old variant call entries, this way, it's uniform.

—
Reply to this email directly or view it on GitHub<https://github.com/ICGC-TCGA-PanCancer/pcawg-central-index/pull/55/files#r49374703>.